test/Microsoft.NET.Publish.Tests/GivenThatAPublishedDepsJsonShouldContainVersionInformation.cs

```diff
@@ -121,135 +121,5 @@ private void Versions_are_included(bool build, [CallerMemberName] string calling
             var depsFilePath = Path.Combine(outputDirectory.FullName, $"{testProject.Name}.deps.json");
             CheckVersionsInDepsFile(depsFilePath);
         }
-
-        [Fact]
```

Comment:
@v-wuzhai were these tests not able to be retargeted to a later framework? They don't look like they are targeting the old tooling technology. Everything else in this PR looks right to me. Our plan is to go into main with this so it shouldn't risk any 0 branches but should improve build times and reliability in 10. We can potentially backport if things look good.